x/payments/handler.go

```diff
@@ -175,7 +198,22 @@ func handleMsgCreatePaymentContract(ctx sdk.Context, k Keeper, bk bank.Keeper,
 	// Submit payment contract
 	k.SetPaymentContract(ctx, contract)
 
-	return sdk.Result{}
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
```

Comment:
We are missing the CanDeauthorise attribute here.

Reply:
Ok . Added the key